frontend/awx/access/organizations/OrganizationPage/OrganizationPage.tsx

```diff
@@ -50,7 +50,7 @@ export function OrganizationPage() {
         icon: EditIcon,
         label: t('Edit organization'),
         onClick: (organization) =>
-          history(RouteObj.EditOrganization.replace(':id', organization?.id.toString() ?? '')),
+          nagivate(RouteObj.EditOrganization.replace(':id', organization?.id.toString() ?? '')),
```

Comment:
should be 'navigate'

Reply:
`useNavigate` hook is set to the variable that is misspelled  as`nagivate`